package.json

```diff
@@ -39,10 +39,12 @@
     "blocked": "^1.1.0",
     "denodeify": "^1.2.1",
     "dotenv": "^1.2.0",
+    "exists-sync": "0.0.3",
```

Comment:
This seems almost as inconsequential as left-pad.  As long as it doesn't have lots of dependencies, fine.